File: inspect_excel.js
const XLSX = require('xlsx');
const path = require('path');

const filePath = path.join('c:', 'Users', 'admin', 'Documents', 'GIT PROJECTS', 'ARTA', 'SRS HR COPY.xlsx');

try {
    const workbook = XLSX.readFile(filePath);
    console.log('Sheet names:', workbook.SheetNames);
    workbook.SheetNames.forEach(sheetName => {
        console.log(`\n--- ${sheetName} ---`);
        const worksheet = workbook.Sheets[sheetName];
        const data = XLSX.utils.sheet_to_json(worksheet, { header: 1 }).slice(0, 10);
        console.log(JSON.stringify(data, null, 2));
    });
} catch (error) {
    console.error('Error:', error.message);
}

File: inspect_excel.py
import pandas as pd

file_path = r'c:\Users\admin\Documents\GIT PROJECTS\ARTA\SRS HR COPY.xlsx'
try:
    xl = pd.ExcelFile(file_path)
    print(f"Sheet names: {xl.sheet_names}")
    for sheet in xl.sheet_names:
        print(f"\n--- {sheet} ---")
        df = pd.read_excel(file_path, sheet_name=sheet).head(10)
        print(df.to_string())
except Exception as e:
    print(f"Error: {e}")
